src/dynarec/rv64/dynarec_rv64_660f.c

```diff
@@ -107,7 +107,34 @@ uintptr_t dynarec64_660F(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int
             SD(x3, gback, 0);
             SD(x4, gback, 8);
             break;
-
+        case 0x72:
+            nextop = F8;
+            switch((nextop>>3)&7) {
+                case 2:
+                    INST_NAME("PSRLD Ex, Ib");
+                    GETEX(x1, 1);
+                    u8 = F8;
+                    if(u8) {
+                        if (u8>31) {
+                            // just zero dest
+                            SD(xZR, x1, 0);
```

Comment:
You need to use `fixedaddress+0` and `fixedaddress+8` for EX

Reply:
Yes, sorry.

Reply:
Fixed.